repl/src/App.js

```diff
@@ -118,6 +119,29 @@ function App() {
     return () => window.removeEventListener('keydown', handleKeyPress);
   }, [pattern, code, activateCode, cycle]);
 
+  useEffect(() => {
```

Comment:
looks good! we could also move this to a custom hook like `useHighlights({pattern,view,started})` or similar, then we could reuse it in the mini repl